src/applications/facility-locator/constants/index.js

```diff
@@ -35,23 +26,25 @@ export const FacilityType = {
  * Enum for map pins.
  * Location types mapped to the filename prefix for the png/svg.
  */
-/* eslint-disable camelcase */
 export const PinNames = {
-  va_health_facility: 'health',
-  cc_provider: 'cc-provider',
-  va_cemetery: 'cemetery',
-  va_benefits_facility: 'benefits',
-  vet_center: 'vet-centers',
+  [FacilityType.VA_HEALTH_FACILITY]: 'health',
+  [FacilityType.VA_CEMETARY]: 'cemetery',
+  [FacilityType.VA_BENEFITS_FACILITY]: 'benefits',
+  [FacilityType.VET_CENTER]: 'vet-centers',
+  [LocationType.CC_PROVIDER]: 'cc-provider',
 };
-/* eslint-enable camelcase */
 
 /**
  * Defines the options available for the Location Type Dropdown
  */
-// TODO: Remove feature flag when going live
-export const LOCATION_OPTIONS = ccLocatorEnabled()
-  ? ['all', 'health', 'cc_provider', 'benefits', 'cemetery', 'vet_center']
-  : ['all', 'health', 'benefits', 'cemetery', 'vet_center'];
+export const LOCATION_OPTIONS = [
+  LocationType.ALL,
+  LocationType.HEALTH,
+  LocationType.CC_PROVIDER,
+  LocationType.BENEFITS,
+  LocationType.CEMETARY,
+  LocationType.VET_CENTER,
+];
```

Comment:
I would rather finish tearing down the feature flag `ccLocatorEnabled` separately, but I had to here because it was causing a circular import.